src/modules/target.js

```diff
@@ -93,6 +94,9 @@ export default {
 
       // Force compute the hi-res image in DOM to prevent
       // image flickering while updating src
+      if (this.srcset) {
+        temp.removeAttribute('srcset')
```

Comment:
Can we remove `srcset` from `this.el` before cloning it to`temp` (before line 93)? Just so we don't need to do the double work on `temp`.

Reply:
Yea, much better.